hypervisor/qemu/qemu.go

```diff
@@ -35,17 +34,6 @@ func qemuContext(ctx *hypervisor.VmContext) *QemuContext {
 	return ctx.DCtx.(*QemuContext)
 }
 
-func InitDriver() *QemuDriver {
-	cmd, err := exec.LookPath("qemu-system-x86_64")
```

Comment:
Two options: 
Three new files for arch specific constants as in this  [branch testing](https://github.com/hyperhq/runv/compare/master...pmorjan:testing?diff=split&name=testing) or 

new const declaration in the existing arch specific qemu_<arch>.go files as in this [branch testing2](https://github.com/hyperhq/runv/compare/master...pmorjan:testing2)

Not sure which one to prefer.